<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>button</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menus">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="bigger_actions"/>
    <addaction name="smaller_actions"/>
    <addaction name="normal_actions"/>
   </widget>
   <addaction name="menus"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="bigger_actions"/>
   <addaction name="smaller_actions"/>
   <addaction name="normal_actions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="bigger_actions">
   <property name="icon">
    <iconset resource="myicons.qrc">
     <normaloff>:/icons/bigger-icon.png</normaloff>:/icons/bigger-icon.png</iconset>
   </property>
   <property name="text">
    <string>最大(&amp;M)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="smaller_actions">
   <property name="icon">
    <iconset resource="myicons.qrc">
     <normaloff>:/icons/smaller-icon.png</normaloff>:/icons/smaller-icon.png</iconset>
   </property>
   <property name="text">
    <string>最小</string>
   </property>
  </action>
  <action name="normal_actions">
   <property name="icon">
    <iconset resource="myicons.qrc">
     <normaloff>:/icons/normal-icon.png</normaloff>:/icons/normal-icon.png</iconset>
   </property>
   <property name="text">
    <string>普通</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myicons.qrc"/>
 </resources>
 <connections/>
</ui>
